src/app/layout.tsx

```diff
@@ -1,13 +1,14 @@
+"use client";
```

Comment:
[imo]

いりましたっけ？

Reply:
`use client` を付けている理由が2つあります🫡

### 1.　layout を client component にしないと fetcher 関数がエラーになる
fetcher に `use client` を付与することでエラーは回避できるのですが
コンポーネントでないものを client component にするのはアリなのでしょうか🤔

■ エラー内容
```
error - node_modules/next/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-server.edge.development.js (2063:12) @ resolveModelToJSON
error - Error: Functions cannot be passed directly to Client Components because they're not serializable.
  {fetcher: function}
```


### 2.　公式ドキュメントを参考
[When to use Server vs. Client Components?](https://beta.nextjs.org/docs/rendering/server-and-client-components#when-to-use-server-vs-client-components)
- Keep large dependencies on the server / Reduce client-side JavaScript

に該当するのかと思いまして...。

Reply:
@ShSato4JPN 

https://www.npmjs.com/package/client-only

fetcher でこれを import するとか？
動作未確認ですが。